frontend/homework/7/index.js

```diff
@@ -0,0 +1,40 @@
+// Establish WebSocket connection with server
+const socket = io('http://localhost:3000');
+
+// DOM manipulation: get input and messages elements
+const input = document.getElementById('input');
+const messages = document.getElementById('messages');
+
+// Event listener for form submission
+form.addEventListener('submit', (e) => {
+    e.preventDefault();
+    if (input.value) {
+        socket.emit('chat message', input.value); // Send message to server
```

Comment:
store this in a constant and make sure there are no spaces in 'chat message' as this is an identifier. (Good Practise)